api/controllers/AuthController.js

```diff
@@ -25,49 +25,67 @@ module.exports = {
      * might fail to complete the login and then be confused when they get redirected somewhere unexpected the next time they visit the site. */
     var login_info = {type: req.query.loginType, redirect: req.query.redirect || '/', validation: crypto.randomBytes(32).toString('hex')};
     req.session.validation = login_info.validation;
-    var auth_data = {state: JSON.stringify(login_info), duration: 'permanent', failureRedirect: '/login'};
-    if (req.query.loginType !== 'mod') {
-      auth_data.scope = 'identity';
+    var auth_data = {state: JSON.stringify(login_info)};
+    if (req.query.loginType === 'mod') {
+      auth_data.duration = 'permanent'; //Mods have a permanent access token. Scope is not specified here, so the scope from config/express.js is used.
+    } else {
+      auth_data.scope = 'identity'; //Regular users only need a temporary access token with reduced scope.
```

Comment:
Right. now that we only request `identity` scope from users there's not much to call anyway.

The scope for the mod is that list in `config/express.js`. This is like what we had before, where the the normal-user login overrides the `default` scopes we have listed there.